File: Java/Language Features/Class Inheritance/src/com/prasadhonrao/inheritance/Employee.java
package com.prasadhonrao.inheritance;
public class Employee extends Person{
    private int employeeId;

    public Employee(int id, String name) {
        this.SetId(id);
        this.SetName(name);
    }

    public String GetEmployeeInformation() {
        String value = this.GetId() + " " + this.GetName();
        return  value;
    }
}

File: C#/Language Features/LINQ/Deferred Execution/Employee.cs
namespace Training.CSharp
{
    public class Employee
    {
        public int Id { get; private set; }
        public string Name { get; private set; }

        public Employee(int id, string name)
        {
            this.Id = id;
            this.Name = name;
        }
    }
}
